sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0b2 (2020-10-06)
+- Updated `azure-communication-chat` version
```

Comment:
```suggestion
- Updated `azure-communication-chat` version.
```

We can add the full-stop at the end everywhere, I forgot nit-picking that in other languages.

(Before GA we'll probably merge all Beta bullets into a single version anyway)